projects/common/models/src/application-base.model.ts

```diff
@@ -0,0 +1,12 @@
+import {UUID} from 'angular2-uuid';
+import {MspImage} from './msp-image';
+
+export interface ApplicationBase {
```

Comment:
This file should be deleted and not in the application as it's specific to MSP only.